monkestation/code/modules/uplink/uplink_items/species.dm

```diff
@@ -9,17 +9,23 @@
 	restricted_species = list(SPECIES_MOTH)
 	surplus = 0
 
-/datum/uplink_item/race_restricted/monkey_barrel
+/datum/uplink_item/species_restricted/monkey_barrel
 	name = "Angry Monkey Barrel"
 	desc = "Expert Syndicate Scientists put pissed a couple monkeys off and put them in a barrel. It isn't that complicated, but it's very effective"
 	cost = 7
 	item = /obj/item/grenade/monkey_barrel
 	restricted_species = list(SPECIES_SIMIAN)
 
-/datum/uplink_item/race_restricted/monkey_ball
+/datum/uplink_item/species_restricted/monkey_ball
 	name = "Monkey Ball"
 	desc = "Stolen experimental MonkeTech designed to bring a monkey's speed to dangerous levels."
 	cost = 12
 	item = /obj/vehicle/ridden/monkey_ball
 	restricted_species = list(SPECIES_SIMIAN)
 
+/datum/uplink_item/species_restricted/tribal_claw_scroll
+	name = "Silver-Scale Scroll"
+	desc = "A scroll with ancient heritage. It can teach the user the secrets of Tribal Claw, an offensive martial art reliant on one's claws and tail."
+	cost = 10
```

Comment:
I would always carp over this at this price, I would say make it a bit cheaper

Reply:
switched to 7 which is the original price if i recall